NthKey.xcodeproj/project.pbxproj

```diff
@@ -975,7 +975,7 @@
 				DEVELOPMENT_TEAM = 227MFQPYQ4;
 				ENABLE_PREVIEWS = YES;
 				INFOPLIST_FILE = "$(SRCROOT)/NthKey/SupportingFiles/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.1;
```

Comment:
This breaks the Travis test, maybe another place needs the version bump too? https://travis-ci.com/github/Sjors/nthkey-ios/builds/226255791#L1477

Reply:
Also, if you're bumping the iOs version, let's also update the recommended project settings and fix the deprecation warnings.

Reply:
fixed in 5b33af1b0cb99302d813be9ee138140f1ad818d8